01-what-is.asciidoc

```diff
@@ -2,4 +2,24 @@
 [[whatis_chapter]]
 == What is the Lightning Network?
 
-Initial paragraph
+Think of a thunderstorm, where electricity is ionizing channels of air to dissipate excess energy and neutralize voltage differences that have been building up during the storm. 
+A channel of ionized air forms when the voltage or potential difference is greater than the breakdown voltage (..kV) between the different regions in the cloud. 
+As the electric current makes its way through the fractal network of ionized air channels it chooses the path of least resistance to reach its destination, a path is formed between the charged up cloud and the ground of earth below. 
+Then a bright flash of light followed by a loud bang occurs as the current dissipated in the path of ionized air channels creates a hot plasma.
+The Lightning Network is named after this natural phenomenon, where the voltage differences are channel balances and the current is a stream of bitcoin of which ownership is transmitted through those channels.
+When lightning strikes the ground, the potential differences are settled at the cost of dissipating energy and making a loud bang.
+In the Lightning Network this happens when payment channels close, the final state of the channel is transmitted to the Bitcoin network with a fee for the miners to include it in the next block.
+
+
+== Why use the Lightning Network?
+
+A second layer protocol can magnify the amount of transactions possible on the settlement layer, so that the base layer will not be overloaded with transactions as Bitcoin gains more active (transacting) participants. 
+Instead small value payments can be instantly transmitted in a Lightning Channel, or a chain of channels through the Lightning Network.
+As these channels are not settled on-chain with every transaction, it will become more difficult to differentiate the smaller payments that occured on the channel from eachother.
```

Comment:
typo

```suggestion
As these channels are not settled on-chain with every transaction, it will become more difficult to differentiate the smaller payments that occured on the channel from each other.
```

Reply:
Also I'm not entirely sure it's correct.